### ISSUE TITLE
Too short nicknames in Facebook buddylist.

### ISSUE THREAD

**Issue by tgielbutowicz** [cryptocat/cryptocat#706]
I have friends on the list with the same name, moreover, their names are too long, so I'm not sure to whom I am writing the message. I would like to be able, to see full contact names on the list.

**Comment by geekman-rohit** [comment]
This is true, there has to be some way to see full name on mouseover the contact name
